boards/arm/npcx7m6fb_evb/doc/index.rst

```diff
@@ -0,0 +1,98 @@
+.. _npcx7m6fb_evb:
+
+NPCX7M6FB_EVB
+###################
+
+Overview
+********
+
+The NPCX7M6FB_EVB kit is a development platform to evaluate the
+Nuvoton NPCX7 series microcontrollers. This board needs to be mated with
```

Comment:
What does mated mean? I don't see a separate board in what you sent over

Reply:
It doesn't mean the separate board. In NPCX series naming role, the 'M' in NPCX7M6FB means package option. For example, '9' in NPCX796FB means 144-balls VFBGA package, '8' means 128-TQFP package,, and so on. The EVB is mated with 144-balls VFBGA package.